src/couch/test/eunit/couchdb_file_compression_tests.erl

```diff
@@ -149,21 +152,10 @@ populate_db(Db, NumDocs) ->
                 ]}
             )
         end,
-        lists:seq(1, 500)
+        lists:seq(1, 100)
     ),
     {ok, _} = couch_db:update_docs(Db, Docs, []),
-    populate_db(Db, NumDocs - 500).
-
-setup_db(SrcDbName) ->
-    TgtDbName = ?tempdb(),
-    TgtDbFileName = binary_to_list(TgtDbName) ++ ".couch",
-    couch_util:with_db(SrcDbName, fun(Db) ->
-        OldPath = couch_db:get_filepath(Db),
-        NewPath = filename:join(filename:dirname(OldPath), TgtDbFileName),
-        {ok, _} = file:copy(OldPath, NewPath)
-    end),
```

Comment:
Not sad to see this go, whatever it's supposed to be doing!

Reply:
This an attempt to avoid uploading 1000 docs to the db again. So it copies the whole file and renames it. But in an effort to save time and perhaps DRY the code it made it more verbose and complicated. Tests are one play where simpler and repetitive duplication can be easier to understand than something very abstracted and complicated.

Reply:
Ah, got it. In that case, I think it could have used a more descriptive name, and perhaps a few comments along the lines of your explanation ;)